### ISSUE TITLE
data-featherlight-filter on data-featherlight-gallery breaks link functionality

### ISSUE THREAD

**Issue by cbaur** [noelboss/featherlight#235]
I've got a div with the data-featherlight-gallery and data-featherlight-filter set to a certain class. In that div, I have some links which hold that class and others which don't. 

The links with the class work as expected (the lightbox shows up) but all the other links (those without the class) stop working - nothing happens when they are clicked.

Fiddle showing the issue: https://jsfiddle.net/wuu5Lcop/1/

**Comment by timstermatic** [comment]
I had this issue too and I could never figure out why it was happening. However, you can highjack it this way:

    $.featherlight.prototype.beforeOpen = function(event) {
        if(!$(event.target).hasClass('your-fl-filter-class')) {
                window.location = $(event.target).attr('href');
                return false;
        }
    }

**Comment by marcandre** [comment]
Oh, looks like a bug with the auto-bind functionality. If you don't use `data-featherlight` and with javascript it'll be ok.
I'll try to fix that asap.

**Comment by dbworth** [comment]
Yes, I also noticed this same bug.  
It can be seen in cbaur's example: https://jsfiddle.net/wuu5Lcop/1/

I took a look at the js code but I couldn't find the problem.

I did find a hack to exclude some images from a gallery, using HTML.   
But I don't know a way to exlude links to external websites. They are broken.
It could be a useful feature to add e.g. featherlight-data="ignore" would force the link to behave in a normal way. 

In this example, images 1 and 2 will be part of the gallery.  
But image 3 will open in a single lightbox, despite being contained within the `<section>` block.
```
<section data-featherlight-gallery="" data-featherlight-filter=".gallery">
<a href="http://www.website.com/image1.jpg" class="gallery">image 1</a>
<a href="http://www.website.com/image2.jpg" class="gallery">image 2</a>
<a href="http://www.website.com/image3.jpg" data-featherlight="">image 3</a>
<a href="http://www.yahoo.com">yahoo</a>
</section>
```
However, unfortunately the link to the external website is broken and doesn't work.

**Comment by marcandre** [comment]
Oh, sorry, forgot about this bug.

Now fixed in 1.7.13.

Thanks for the great bug report.
